hack/install-kind.sh

```diff
@@ -13,11 +13,13 @@ set -eu
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 
 # kind version
-KIND_VERSION="${KIND_VERSION:-v0.22.0}"
+KIND_VERSION="${KIND_VERSION:-v0.24.0}"
```

Comment:
I would not do this without further changes. While it in most cases works, KinD is recommending to use Kubernetes KinD node images that were built for a specific KinD version (see their release notes: https://github.com/kubernetes-sigs/kind/releases/tag/v0.24.0). That means we would have to adjust the used images at other places as well.

Though, bumping Kubernetes dependencies is on our list anyway in preparation of the next Shipwright release.

Reply:
Thanks for your attention @SaschaSchwarze0 , I found the script in `try it` section of the `README`, as part of setting up the shipwright I run the install-kind.sh script and found it failed without successfully installing the Kind clusters. Hence I created the issue and a PR against it. I agree with your point. I can close this issue and the PR now.